src/nicolai_munsterhjelm_ex/ex02/bubble_sort.py

```diff
@@ -0,0 +1,32 @@
+def bubble_sort(data):
+    """This function sorts a tuple or list of numbers using the bubble sort algorithm.
+
+    Parameters
+    ----------
+    data (tuple or list)
+        The tuple/list to be sorted.
+
+    Returns
+    -------
+    new_list
+        Sorted list.
+    """
+
+    new_list = list(data)
+
+    for i in range(len(new_list) - 1):
```

Comment:
consider creating a `sorting_length = len(data) + 1` and using that in the `range() object`